src/pods/canvas/canvas.pod.tsx

```diff
@@ -80,14 +79,15 @@ export const CanvasPod: React.FC = () => {
 
   const containerRef = React.useRef<HTMLDivElement>(null);
 
-  const handleScroll = () => {
-    if (containerRef.current) {
-      setScrollPosition({
-        x: containerRef.current.scrollLeft,
-        y: containerRef.current.scrollTop,
-      });
-    }
-  };
+  // const handleScroll = () => {
+  //   if (containerRef.current) {
+  // setPosition comes from the useCanvasViewSettingsContext
+  //     setScrollPosition({
+  //       x: containerRef.current.scrollLeft,
+  //       y: containerRef.current.scrollTop,
+  //     });
+  //   }
+  // };
```

Comment:
Should this code be removed if it's commented? Or explain why is commented

Reply:
It's fixed. I forgot what the function does.

Reply:
Now the add collection and place the table where the scroll is is broken, can you take a look?

Reply:
Fixed, something happened with my last commit.